packages/genesys-spark-components/src/components/beta/gux-date-time/gux-date-time.tsx

```diff
@@ -45,7 +46,7 @@ export class GuxDateTime {
 
   componentWillLoad(): void {
     trackComponent(this.root);
-    if (readRegionalDatesCookie()) {
+    if (readRegionalDatesCookie() || readRegionalDatesVar()) {
```

Comment:
consider abstracting this to an individual function that check both sources. `useRegionalDates`